documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -22,10 +22,10 @@ SQL based polling model of tables put into "capture mode". This uses the ASN lib
 In order to use ASN and hence this connector, you need to have a license for the IBM InfoSphere Data Replication (IIDR) product.
 It is not required that IIDR itself is installed.
 
-The first time it connects to a Db2 database, it reads a consistent snapshot of all of the tables that are whitelisted (or not blacklisted depending on the mode of operation).
```

Comment:
No mention of the actual properties for Db2?

Reply:
If so indeed, can you please log an issue and assign it to @TovaCohen? Thx!

Reply:
DB2 is in the incubator repo. I will need to update incubator connectors in another PR. What should in general happen to incubator connectors are they always pointing to the recent DBZ version or frozen to a specific (maybe older) version until such changes on core are applied in the incubator repo as well?

Reply:
maybe we should reason about moving incubator to a directory in this repo and have it mono-repo-ed? would make such changes easier.

Reply:
Usually we send two PRs, stating on the incubator one that it requires the core PR xzy.

Reply:
Note my question was about those props apparently being undocumented for Db2. If you can confirm this, can you please log an issue? Thx!

Reply:
misunderstanding, sorry! with my comment I meant that they are documented, but I didn't update the config yet because I wasn't sure on how we handle incubator connectors. I will quickly change the docs and create another PR for incubator later this week to actually change the incubator code accordingly.

Reply:
Ah, gotcha now. Your call whether you'd do the doc update for the incubator connectors as part of this PR, or send another PR for that alongside the incubator PR for the actual code changes.

Reply:
I am currently working through the Db2 connector doc. I completely rewrote the sentence that this comment thread is about. The sentence is now: "When the connector starts, it performs a consistent snapshot] of the Db2 database tables that the connector is configured to capture changes for. "  So you don't need to update the Db2 doc. Okay?

Reply:
Thank you @TovaCohen!

@gunnarmorling I decided to do the incubator connector changes first because the old properties will still be supported and I can't estimate how much time I will need to do the incubator connectors. It would be bad if I change the docs now and it will take +1 week to get the incubator changes out of the door and people read docs that are not up to date with code changes.